# References Used

1. Asgari E, Mofrad MRK (2015) Continuous Distributed Representation of Biological Sequences for Deep Proteomics and Genomics. PLOS  ONE 10(11): e0141287. https://doi.org/10.1371/journal.pone.0141287

2. Chen X, Yan CC, Zhang X, Zhang X, Dai F, Yin J, et al. Drug–target interaction prediction: databases, web servers and computational models. Briefings in Bioinformatics. 2015;17: 696–712. doi:10.1093/bib/bbv066 

3. Jaeger S, Fulle S, Turk S. Mol2vec: Unsupervised Machine Learning Approach with Chemical Intuition. Journal of Chemical Information and Modeling. 2018;58: 27–35. doi:10.1021/acs.jcim.7b00616

4. Olayan RS, Ashoor H, Bajic VB. DDR: efficient computational method to predict drug–target interactions using graph mining and machine learning approaches. Bioinformatics. 2017;34: 1164–1173. doi:10.1093/bioinformatics/btx731

5. Pahikkala T, Airola A, Pietila S, Shakyawar S, Szwajda A, Tang J, et al. Toward more realistic drug-target interaction predictions. Briefings in Bioinformatics. 2014;16: 325–337. doi:10.1093/bib/bbu010 

6. Wen M, Zhang Z, Niu S, Sha H, Yang R, Yun Y, et al. Deep-Learning-Based Drug–Target Interaction Prediction. Journal of Proteome Research. 2017;16: 1401–1409. doi:10.1021/acs.jproteome.6b00618 

7. Yamanishi Y. Supervised Bipartite Graph Inference. Advances in Neural Information Processing Systems 21 (NIPS 2008). Neural Information Processing Systems Conference;
